boards/arm/colibri_imx7d_m4/colibri_imx7d_m4.dts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2017, NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_imx.dtsi>
```

Comment:
Please adjust the include after rename.

Reply:
I have changed it in my local repo and for some reason the "boards/arm: add support for colibri_imx7d_m4 board " commit was not pushed to my zephry repo fork.

Reply:
I think I forgot the last "git rebase --continue".